README.md

```diff
@@ -12,7 +12,9 @@ Projects listed on this page are open source that Microsoft maintain or contribu
 |                                      | [Cluster API Azure Provider](https://sigs.k8s.io/cluster-api-provider-azure) <br> ([releases](https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases)) <br> [Tests](https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-azure)                      | Self-managed clusters on Azure using Cluster API                                                                                                                                                                                                                              | CNCF: incubating <br> API: v1alpha4                                                                                                                                                                    | [#cluster-api-azure](https://kubernetes.slack.com/archives/CEX9HENG7) <br> [kubernetes-sig-cluster-lifecycle@googlegroups.com](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) <br> [GitHub issues](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues)                                                                                            |                                                                                                                                                                                                                                                                                                                               |
-|                                      | [AKS Engine](https://github.com/Azure/aks-engine) <br> ([releases](https://github.com/Azure/aks-engine/releases))                                                                                                                                                                  | Self-managed clusters on Azure                                                                                                                                                                                                                                                | Azure: Deprecating. Consider using [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) or [Cluster API Azure Provider](https://sigs.k8s.io/cluster-api-provider-azure) <br> API: N/A | [#aks-engine-users](https://kubernetes.slack.com/archives/CU3N85WJK) <br> [GitHub issues](https://github.com/Azure/aks-engine/issues)                                                                                                                                                                                                                                                          | <li>[Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-kubernetes-aks-engine-overview)</li>                                                                                                                                                                                                      |
```

Comment:
Instead of deleting the projects that have been deprecated/archived, could you add a new table `Past Projects` and move this there? It doesn't have to be in the main readme, could just be another markdown so we maintain history.

cc @sozercan @ritazh @bridgetkromhout

Reply:
Done